src/components/AirTicketsSearch/AirTicketsSearch.component.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import DateRangePicker from 'react-bootstrap-daterangepicker';
+import 'bootstrap-daterangepicker/daterangepicker.css';
+
+import ItineraryModalContainer from '../ItinerarayModalContainer/ItineraryModalContainer.component';
+import PassengSubContent from '../PassengerSubContent/PassengerSubContent.component';
+
+import {
+	AirTicketsSearchContainer,
+	SearchContainer,
+} from './AirTicketsSearch.styles';
+
+const AirTicketsSearch = () => {
+	const [passengerSubContentHidden, setPassengerSubContentHidden] = useState(
+		true,
+	);
+	const [adultAmount, setAdultAmount] = useState(0);
+	const [childAmount, setChildAmount] = useState(0);
+	const [babyAmount, setBabyAmount] = useState(0);
+	const [departure, setDeparture] = useState('김포');
+	const [arrival, setArrival] = useState('제주');
+	const [modalHidden, setModalHidden] = useState(true);
+	const [seatSelection, setSeatSelection] = useState([
+		true,
+		false,
+		false,
+		false,
+	]);
+
+	const [locationTarget, setLocationTarget] = useState(null);
+
+	const changeLocation = (e) => {
+		if (locationTarget === departure) setDeparture(e.target.innerText);
+		else setArrival(e.target.innerText);
+		setModalHidden(!modalHidden);
+	};
+
+	const getSelection = (num) => {
+		let selectionArray = [false, false, false, false];
+		selectionArray[num] = true;
+		setSeatSelection(selectionArray);
+	};
```

Comment:
```suggestion
const getSelection = (num) => {
  const arr = Array(4).fill().map((_, idx) => idx === num)
  
  setSeatSelection(arr);
};
```
연습 삼아서라도 배열 메서드 의식적으로 활용해주세요

Reply:
수정했습니다 😄